Sources/XiEditor/RPCSending.swift

```diff
@@ -44,41 +44,6 @@ enum RpcResult {
     case ok(AnyObject)
 }
 
-enum RPCRequestMethod: String {
-    case measureWidth = "measure_width"
-}
-
-enum RPCNotificationMethod: String {
```

Comment:
These have now become `CoreRequest` and `CoreNotification`. This was partly to give me a migration path as I built out the new parsing approach. I'd be happy to rename them back to their `RPC...` versions if needed.